README.md

```diff
@@ -3063,6 +3063,7 @@ _Full stack web frameworks._
 - [rex](https://github.com/goanywhere/rex) - Rex is a library for modular development built upon gorilla/mux, fully compatible with `net/http`.
 - [rk-boot](https://github.com/rookie-ninja/rk-boot) - A bootstrapper library for building enterprise go microservice with Gin and gRPC quickly and easily.
 - [rux](https://github.com/gookit/rux) - Simple and fast web framework for build golang HTTP applications.
+- [sesh](https://github.com/joshmedeski/sesh) - Sesh is a CLI that helps you create and manage tmux sessions quickly and easily using zoxide.
```

Comment:
Ohh, may be a mistake but why is it listed as a web framework. ❓

Reply:
oof, didn't realise, I searched for `tmux` and found one in utitiles to make life easier. and must've hit next.

Reply:
fixed